constructors.js

```diff
@@ -10,15 +10,33 @@
  * @property {string} description
  * @method   printDetails
  */
-
+  function Spell (name, cost, description) {
+    if (typeof(name) === 'string') {
+      this.name = name;
+    } else {
+      throw new TypeError('Spell name must be a string.');
+    }
+    if (typeof(cost) === 'number') {
+      this.cost = cost;
+    } else {
+      throw new TypeError('Spell cost must be a number.');
+    }
+    if (typeof(description) === 'string') {
+      this.description = description;
+    } else {
+      throw new TypeError('Spell description must be a string.');
+    }
   /**
    * Returns a string of all of the spell's details.
    * The format doesn't matter, as long as it contains the spell name, cost, and description.
    *
    * @name getDetails
    * @return {string} details containing all of the spells information.
    */
-
+  this.getDetails = function () {
```

Comment:
Make sure you are adding these to the `Spell.prototype` instead of as instance methods.